.changeset/grumpy-ads-know.md

```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/compiler": patch
+---
+
+Allows conditional rendering of scripts when `RenderScript` is enabled
```

Comment:
```suggestion
Fixes an issue with the conditional rendering of scripts. 

**This change updates a v5.0 breaking change when `experimental.directRenderScript` became the default script handling behavior.** If you have already successfully upgraded to Astro v5, you may need to review your script tags again and make sure they behave as desired. [See the v5 Upgrade Guide for more details.](https://docs.astro.build/en/guides/upgrade-to/v5/#script-tags-are-rendered-directly-as-declared)
```

So, just a suggestion, but something of this style maybe? (And, check the content at the upgrade guide link to see whether there's anything else here needed?)

Reply:
Yeah this is great!